### PR TITLE
fix landing page dark theme Background pattern

### PR DESCRIPTION
**Pull request by TheIsotopes**
should now be displayed with the correct background color. thx @KarateBrot to found this.

image

### CODE REVIEW

File: src/css/dark-theme.less [hunk 1]
```diff
@@ -395,8 +395,7 @@ progress[value] {
     }
 }
 .tab-landing {
-    .content_top {
```
Comment (haslinghuis) [inline src/css/dark-theme.less:398]: Also remove the other bracket

File: src/css/dark-theme.less [hunk 2]
```diff
@@ -395,8 +395,7 @@ progress[value] {
...
-        background-color: #00000000;
+    background-color: #3e3e3e;
     }
```
Comment (haslinghuis) [inline src/css/dark-theme.less:399]: Think you need to remove this one?

File: src/css/dark-theme.less (hunk 1 above)
Comment (TheIsotopes) [reply src/css/dark-theme.less:398]: fixed now.

File: src/css/dark-theme.less (hunk 2 above)
Comment (haslinghuis) [inline src/css/dark-theme.less:399]:
```suggestion

```

File: src/css/dark-theme.less [hunk 3]
```diff
@@ -409,7 +408,6 @@ progress[value] {
     .logowrapper {
         img {
             content: url(../images/bf_logo_black.svg);
-        }
```
Comment (haslinghuis) [inline src/css/dark-theme.less:412]: This one needs to stay

**Review by haslinghuis — APPROVED** [review]

File: src/css/dark-theme.less (hunk 3 above)
Comment (TheIsotopes) [reply src/css/dark-theme.less]: now it should be fine

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by blckmn — APPROVED** [review]

**Pull request merged**